web/src/components/NavigationTopbar.vue

```diff
@@ -19,15 +19,73 @@
           />
       </b-navbar-brand>
     </b-navbar>
-    
+   
+    <div style="display: flex; flex-direction: row; background-color: grey;">
+      <div style="width: 80%;">
+        <ul>
+          <li>
+            <a class="nav-link" href="/Home">Dashboard</a>
+          </li>
+          <li>
+            <a class="nav-link">Court Calendar</a>
+          </li>
+          <li>
+            <a class="nav-link">Court List</a>
+          </li>
+          <li>
+            <a class="nav-link">Court File Search</a>
+          </li>
+          <li>
+            <a class="nav-link">DARS</a>
+          </li>
+        </ul>
+      </div>
+      <div style="width: 20%;">
+        <ul>
+          <li>
+            <a class="nav-link">Settings</a>
+          </li>
+          <li>
+            <a class="nav-link">Quick Links</a>
+          </li>
+        </ul> 
+      </div>
+    </div>
+    <h2>Welcome {{ userTitle }}</h2>
   </header>
 </template>
 
+<style lang="scss">
+  @import "../styles/header";
+</style>
+
 <script lang="ts">
-  import { Component, Vue } from 'vue-property-decorator';
+import { UserInfo } from '@/types/common';
+import "@store/modules/CommonInformation";
+import { Component, Vue } from 'vue-property-decorator';
+import { namespace } from "vuex-class";
+const commonState = namespace("CommonInformation");
+
+@Component
+export default class NavigationTopbar extends Vue {   
+
+  @commonState.State
+  public userInfo!: UserInfo;
+
+    userName = '';
+    userRole = '';
+    userTitle = '';
 
-  @Component
-  export default class NavigationTopbar extends Vue {   
+    mounted(){
+      this.getUserInfo();
+    }
 
+    public getUserInfo(): void{
+      // TODO: Get user information so this can be populated properly
```

Comment:
I found that we already have some kind of user info stored in the state [here](https://github.com/bcgov/jasper/blob/7b567d6a5e768e26262997d2a63fcf7cf8f193f1/web/src/utils/utils.ts#L7) but the endpoint may need to be adjusted to add other properties. May be we can add this as part of the TODO comment so the next developer has a starting point.

Reply:
Resolving the comment since there is a ticket https://jag.gov.bc.ca/jira/browse/JASPER-141 that covers backend details where to get the user info.